src/ngx_http_lua_ssl_certby.c

```diff
@@ -1236,6 +1360,40 @@ ngx_http_lua_ffi_set_priv_key(ngx_http_request_t *r,
 }
 
 
+int
+ngx_http_lua_ffi_set_httpv(ngx_http_request_t *r,
```

Comment:
The `httpv` abbreviation looks weird. How about the name `ngx_http_lua_ffi_use_http2` to make our intention clearer here?